webdriver/commands/AcceptAlert.json

```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/WebDriver/Commands/AcceptAlert",
           "support": {
             "chrome": {
-              "version_added": false,
+              "version_added": "65",
```

Comment:
Mostly because this is where they started supporting W3C spec commands. I am happy for Google to take this on to collect a more correct value but seeing as people are less likely to use that version and more likely to use 73 now so shouldn't really care.

Reply:
I'm comfortable leaving this at 65. If I understand this correctly, there's a difference between landing and Chrome and landing in [ChromeDriver](http://chromedriver.chromium.org/). I'm guessing the latter is what we care about here.

FYI, here's a [current support list](https://chromium.googlesource.com/chromium/src/+/HEAD/docs/chromedriver_status.md). It doesn't tell you when anything lands, but at least we can make sure there are no null values in the data.